flytekit/bin/entrypoint.py

```diff
@@ -458,7 +456,7 @@ def execute_task_cmd(
         _click.echo("No resolver found, assuming legacy API task...")
         _legacy_execute_task(task_module, task_name, inputs, output_prefix, raw_output_data_prefix, test)
     else:
-        _click.echo(f"Attempting to run with {resolver}...")
+        logger.debug(f"Running task execution with resolver {resolver}...")
```

Comment:
why not replace all uses of `_click.echo` in this file?

Reply:
not sure, it's still a click cli at the end of the day.  should we?

Reply:
changed a couple more.